webpack/set-webpack-public-path-plugin/src/SetPublicPathPlugin.ts

```diff
@@ -2,11 +2,12 @@
 // See LICENSE in the project root for license information.
 
 import { EOL } from 'os';
-import type * as Webpack from 'webpack';
-import type * as Tapable from 'tapable';
+import { VersionDetection } from '@rushstack/webpack-plugin-utilities';
 
+import type * as Webpack from 'webpack';
 // Workaround for https://github.com/pnpm/pnpm/issues/4301
 import type * as Webpack5 from '@rushstack/heft-webpack5-plugin/node_modules/webpack';
+import type * as Tapable from 'tapable';
```

Comment:
```suggestion
import type * as Webpack from 'webpack';
import type * as Tapable from 'tapable';
// Workaround for https://github.com/pnpm/pnpm/issues/4301
import type * as Webpack5 from '@rushstack/heft-webpack5-plugin/node_modules/webpack';
```

Just to minimize the diff.